astropy/visualization/tests/test_histogram.py

```diff
@@ -3,10 +3,10 @@
 
 from numpy.testing import assert_allclose
 
-from astropy.utils.compat.optional_deps import HAS_PLT, HAS_SCIPY
+from astropy.utils.compat.optional_deps import HAS_MATPLOTLIB, HAS_SCIPY
```

Comment:
I don't understand this change.

https://github.com/astropy/astropy/blob/e15110df668513d282c337660c6fbb1a13280c18/astropy/utils/compat/optional_deps.py#L48

Reply:
It's a nit of mine: checking for "matplotlib.pyplot" instead of matplotlib directly feels weird to me (none of our other optional deps are checked this way). I felt that it would be time to clean it up now that the `pyplot` interface is to be actively avoided internally.

Reply:
I feel like it is out of scope. Can we revisit renaming the import as a separate PR, though my preference is just to leave it as-is. Why fix something that isn't broken and no one is complaining about, and it is less typing.

Reply:
> it is less typing.

how about `HAS_MPL` then ?

Reply:
+0 on that. But regardless I don't think we should mix that in with a PR titled "avoid pyplot in visualization tests". I think this PR should purely be about the tests and not changing any import names. Thanks for your understanding!

Reply:
reverted

Reply:
Thanks! I'll have a look in a day or two. :)